test/scripts/sticky/test-sticky-provider-routes.sh

```
@@ -0,0 +1,375 @@
+        rm -f "$headers_file" "$body_file"
+        return
+    fi
+    pass "Turn 1 HTTP ${http_code}"
+
+    local sticky1 ep1 mode1 marker1
+    sticky1=$(extract_header "$headers_file" "X-Olla-Sticky-Session")
+    ep1=$(extract_header "$headers_file" "X-Olla-Endpoint")
+    mode1=$(extract_header "$headers_file" "X-Olla-Mode")
+    marker1=$(extract_backend_marker "$(cat "$body_file")")
+
+    [[ "$sticky1" == "miss" ]] && pass "Turn 1 sticky=miss" || fail "Turn 1 sticky='${sticky1}' (expected miss)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)